pyproject.toml

```diff
@@ -19,14 +19,13 @@ maintainers = [
 ]
 description = "A succinct matplotlib wrapper for making beautiful, publication-quality graphics."
 readme = "README.rst"
-requires-python = ">=3.6.0"
+requires-python = ">=3.8, <3.12"
```

Comment:
Why the upper bound here?

Reply:
basemap won't install for python>=3.12. Eventhough we don't have a hard requirement for it some of the tests will fail. I contacted the people from basemap and they are working on a fix. I put a crude PR up there that monkey patches basemap for these higher versions, but it isn't (and will likely never be) merged. 

I checked the code with this monkey patch for basemap and ultraplot will pass all the tests for these higher versions.